Cargo.toml

```
@@ -17,7 +17,8 @@ repository = "Metaswitch/swagger-rs"
 default = ["serdejson"]
 multipart_form = ["mime"]
 multipart_related = ["mime_multipart", "hyper_0_10", "mime_0_2"]
-serdejson = ["serde", "serde_json"]
+serdejson = ["serde", "serde_json", "serde_valid", "regex", "paste"]
```

I don't think these dependencies should be part of serdejson?

```suggestion
serdejson = ["serde", "serde_json"]
```